output/html/assets_styles.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Assets from '../assets.js';
+import Tabular from '../tabular.js';
+import { __, _x, _n, sprintf } from '@wordpress/i18n';
+
+class Styles extends Component {
```

Comment:
Does it make sense for there to be a Scripts and a Styles component with exactly the same implementation?

Reply:
Currently I'm building the components to match the existing PHP implementations. You're right that with the new model this is probably not needed, but then I haven't fixed up all the labelling yet so we'll see.